lib/addPipelineExtras.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var defaultValue = Cesium.defaultValue;
+
+module.exports = addPipelineExtras;
+
+//Add the pipeline extras object to each glTF object.
+function addPipelineExtras(gltf) {
+    var reference = {

```

Comment:
Does the `reference` object need the `addPipelineExtra` properties?  My original - but perhaps too simplistic - thought was just having the glTF properties name would be enough, for example:
```javascript
var reference = {
    "accessors" : true,
    "animations" : {  // implicitly true since all glTF object properties have an extras sub-property
        "channels" : [ // same comment
            {
                "target" : true
            }
        ],
        "samplers" : true
    }
    // ...
}
```
Does this make the algorithm harder (or impossible) to implement?